version.properties

```diff
@@ -0,0 +1,3 @@
+# Version of the produced binaries.
+# The version is inferred by shipkit-auto-version Gradle plugin (https://github.com/shipkit/shipkit-auto-version)
+version=0.11.*
```

Comment:
Since we are forking Iceberg, this should be the version of Iceberg we forked (`0.11.1`) followed by another level of version (below patch, e.g., `0.11.1.*`). @mockitoguy, Shipkit seems to complain about this. Do you how we can fix it?

Reply:
Modified `shipkit-auto-version` to be 1.2.0 with the fix and `version.properties`, PTAL.

Reply:
Thank you @mockitoguy for the prompt fix!